lte/gateway/c/core/oai/tasks/gtpv1-u/gtp_tunnel_upf.c

```diff
@@ -49,17 +49,17 @@ const struct gtp_tunnel_ops* upf_gtp_tunnel_ops_init_openflow(void) {
 }
 
 int upf_add_tunnel(struct in_addr ue, struct in6_addr* ue_ipv6, int vlan,
-                   struct in_addr enb, uint32_t i_tei, uint32_t o_tei,
-                   Imsi_t imsi, struct ip_flow_dl* flow_dl,
+                   struct in_addr enb, struct in6_addr* unused, uint32_t i_tei,
```

Comment:
can we also update all added unused params with pattern: `unused_<type_of_param>`